assets/js/app.js

```diff
@@ -15,7 +15,9 @@ require('bootstrap');
 // var $ = require('jquery');
 
 window.jQuery = require('jquery');
+window.bootstrap = require('bootstrap');
```

Comment:
Where is this used?

Reply:
1) I will fix it
2) Please provide screenshot

Reply:
Bootstrap removed